samples/snippets/pom.xml

```diff
@@ -61,7 +62,7 @@
     <dependency>
       <groupId>org.apache.beam</groupId>
       <artifactId>beam-model-pipeline</artifactId>
-      <version>2.29.0</version>
+      <version>${beam.version}</version>
     </dependency>
     <!-- [END pubsublite_java_dependencies] -->
```

Comment:
Need to move the region tags to include only `google-cloud-pubsublite` and `google-cloud-pubsublit`, these show up on https://cloud.google.com/pubsub/lite/docs/reference/libraries#client-libraries-install-java, and is for running Lite samples only.

Deps for cross-product samples should not be included by this region tag.

Reply:
Done